DualView/src/main/java/com/microsoft/device/display/samples/dualview/view/MapSimulator.kt

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.device.display.samples.dualview.view
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Matrix
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.toBitmap
+import com.microsoft.device.display.samples.dualview.R
+import com.microsoft.device.display.samples.dualview.model.RestaurantCoordinates
+
+/**
+ * Creates a bitmap that simulates Google Maps. The bitmap is obtained from a background and a set of pins.
+ */
+class MapSimulator(val context: Context) {
+    private val density: Float = context.resources.displayMetrics.density
+    private val initialMap =
```

Comment:
Usually, we are adding the type for properties. Also, when it;s about resources, we are using lazy initialization:
private val initialMap: Bitmap by lazy { 
       ContextCompat.getDrawable(context, R.drawable.unselected_map)?.toBitmap()
}

Reply:
Fixed